etc/README.rst

```diff
@@ -0,0 +1,100 @@
+Availability Service Configuration
+==================================
+
+Placement
+---------
+
+Configuration file should be placed to */etc/oss/availability/config.json*.
+
+If there is an environment variable *AVAILABILITY_CONF* set, it is used as priority.
+In this case it is important to use absolute path to configuration file.
+
+Example:
+
+.. code-block::
+
+    export AVAILABILITY_CONF=/home/username/oss_configs/availability.json
+    python availability/main.py
+
+Configuration File Description
+------------------------------
+
+Configuration file is plain JSON document which defines
+`Flask <http://flask.pocoo.org>`_, connection to data storage and
+services quering configuration:
+
+Here is a simple example:
+
+.. code-block::
+
+  {
+      "flask": {
+          "PORT": 5020,
+          "HOST": "0.0.0.0",
+          "DEBUG": false
+      },
+      "elastic": "http://127.0.0.1:9200",
```

Comment:
I would prefer to change this now. it's not flexible and clear. 

"backend": {
    "type": "elastic",
    "connection": "http://127.0.0.1:9200"
}

this will allow us in future to implemenet different backends without changing of input format.